spacialgaze-plugins/misc.js

```diff
@@ -433,4 +433,25 @@ exports.commands = {
 		Db.disabledScrolls.remove(target);
 	},
 	enableintroscrollhelp: ["/enableintroscroll [room] - Enables scroll bar preset in the room's roomintro."],
+	'!ai': true,
```

Comment:
Just for visual  purposes make an empty line after enableintroscrollhelp's line imo